spring-boot-project/spring-boot-autoconfigure/src/test/java/org/springframework/boot/autoconfigure/task/TaskExecutionAutoConfigurationTests.java

```diff
@@ -451,6 +485,10 @@ void shouldNotAliasApplicationTaskExecutorWhenBootstrapExecutorAliasIsDefined()
 			});
 	}
 
+	private TaskDecorator createTaskDecorator() {
+		return (runnable) -> runnable;
```

Comment:
A custom implementation instead of `Mockito.mock(TaskDecorator.class)` to avoid this exception
```
Exception in thread "scheduling-1" java.lang.NullPointerException: Cannot invoke "java.lang.Runnable.run()" because "this.decoratedRunnable" is null
	at org.springframework.scheduling.concurrent.ThreadPoolTaskScheduler$DelegatingRunnableScheduledFuture.run(ThreadPoolTaskScheduler.java:508)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
	at java.base/java.lang.Thread.run(Thread.java:840)
```